### PR TITLE
bug 1500224 changing playbook for router

### PR DESCRIPTION
**Pull request by kalexand-rh**
Follow up to https://github.com/openshift/openshift-docs/pull/9989

### CODE REVIEW

File: install_config/certificate_customization.adoc [hunk 1]
```diff
@@ -248,7 +248,8 @@ ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/redeploy-certifi
 ----
 
 [NOTE] 
-For this playbook to run, the certificate names must not change. If the certificate names change, rerun the Ansible *_deploy_cluster.yml_* playbook
```
Comment (jianlinliu) [inline install_config/certificate_customization.adoc]: This part is irrelevant to customerized router certs, so no need this change.

Comment (kalexand-rh) [reply install_config/certificate_customization.adoc]: Thank you! Fixed.

File: install_config/certificate_customization.adoc [hunk 2]
```diff
@@ -226,17 +226,17 @@ Where the parameter values are:
...
-If these certificate files are new to your {product-title} cluster, run the Ansible *_deploy_cluster.yml_* playbook to add these files to the {product-title} configuration files. 
-The playbook adds the certificate files to the *_/etc/origin/master/named_certificates/_* directory.
+If these certificate files are new to your {product-title} cluster, run the Ansible *_deploy_router.yml_* playbook to add these files to the {product-title} configuration files. 
+The playbook adds the certificate files to the *_/etc/origin/master/_* directory.
 
 ----
 ifdef::openshift-enterprise[]
 # ansible-playbook [-i /path/to/inventory] \
-    /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
+    /usr/share/ansible/openshift-ansible/playbooks/deploy_router.yml
```
Comment (jianlinliu) [inline install_config/certificate_customization.adoc]: The correct path is "/usr/share/ansible/openshift-ansible/playbooks/openshift-hosted/deploy_router.yml"

File: install_config/certificate_customization.adoc [hunk 3]
```diff
@@ -226,17 +226,17 @@ Where the parameter values are:
...
 endif::[]
 ifdef::openshift-origin[]
 # ansible-playbook [-i /path/to/inventory] \
-    ~/openshift-ansible/playbooks/deploy_cluster.yml
+    ~/openshift-ansible/playbooks/deploy_router.yml
```
Comment (jianlinliu) [inline install_config/certificate_customization.adoc]: The correct path is "~/openshift-ansible/playbooks/deploy_router.yml"

**Comment by kalexand-rh** [comment]
@jianlinliu, I updated the paths per your comments.

File: install_config/certificate_customization.adoc (hunk 3 above)
Comment (jianlinliu) [reply install_config/certificate_customization.adoc]: Sorry to mislead you due to my typo in my last comment, the correct path should be "~/openshift-ansible/playbooks/openshift-hosted/deploy_router.yml"

Comment (kalexand-rh) [reply install_config/certificate_customization.adoc]: @jianlinliu, no worries. I'll fix the path and get the changes ready to merge. :)

**Comment by kalexand-rh** [comment]
Jianlin approved this PR, pending the change I just made, in the source bug. Since it's just path changes, I'm merging without peer review.

**Pull request merged**